src/Umbraco.Web.UI.Client/src/views/propertyeditors/mediapicker3/umbMediaPicker3PropertyEditor.component.js

```diff
@@ -396,6 +397,19 @@
             });
         }
 
+        function validateMandatory() {
+            debugger;
```

Comment:
@ustadstar Make sure to remove the debugger again! 😄

Reply:
@nul800sebastiaan my bad.. I have removed it!